WooCommerce/Classes/Tools/Zendesk/ZendeskManager.swift

```diff
@@ -324,7 +371,10 @@ final class ZendeskManager: NSObject, ZendeskManagerProtocol {
     /// The SDK tag is used in a trigger and displays tickets in Woo > Mobile Apps New.
     ///
     func getTags(supportSourceTag: String?) -> [String] {
-        let tags = [Constants.platformTag, Constants.sdkTag, Constants.jetpackTag]
+        var tags = [Constants.platformTag, Constants.sdkTag, Constants.jetpackTag]
+
+        // Get IPP plugin statuses
+        tags.append(contentsOf: ippPluginstatuses ?? [])
```

Comment:
If we change the type of `ippPluginstatuses` to non-optional, this can be simplified as:
```suggestion
        let tags = [Constants.platformTag, Constants.sdkTag, Constants.jetpackTag] + ippPluginstatuses
```

Reply:
That's right, we'll always be passing a value here so no reason for it to be optional. Changed here: c36468978777c5137a019484e90ea3e7ab561ed7